fedora/qesteidutil.spec

```diff
@@ -0,0 +1,226 @@
+# package does not work in F26 because openssl-devel is 1.1 version
+# but Qt has been built against OpenSSL 1.0 version, so qesteidutil will crash
+# Therefore this spec file is for Fedora >= 27 only
+
+Name:           qesteidutil
+Version:        3.12.10
+Release:        1%{?dist}
+Summary:        Estonian ID card utility
+License:        LGPLv2+
+URL:            https://github.com/open-eid/qesteidutil
+Source0:        %{url}/releases/download/v%{version}/%{name}-%{version}.tar.gz
+
+Source1:        config.json
+Source2:        config.pub
+Source3:        config.rsa
+Source4:        config.qrc
+
+Patch0:         sandbox-compilation.patch
+
+BuildRequires:  cmake
+BuildRequires:  desktop-file-utils
+BuildRequires:  pkgconfig(openssl)
+BuildRequires:  pkgconfig(Qt5Designer)
+BuildRequires:  pkgconfig(libpcsclite) >= 1.7
+BuildRequires:  libappstream-glib
+BuildRequires:  qtsingleapplication-qt5-devel
+Requires:       hicolor-icon-theme
+Requires:       opensc%{?_isa}
+Requires:       pcsc-lite-ccid%{?_isa}
+
+%description
+QEsteidUtil is an application for managing Estonian ID Card. In an
+user-friendly interface it is possible to change and unlock PINs, examine
+detailed information about personal data file on the card, extract and view
+certificates, set up mobile ID, and configure @eesti.ee email.
+
+%prep
+%setup -q
+
+cp %{S:1} common/
+cp %{S:2} common/
+cp %{S:3} common/
+cp %{S:4} common/
+
+# Remove bundled qtsingleapplication to make sure it isn't used
+rm -rf qtsingleapplication
+# COPING is not needed as it is the license for the removed qtsingleapplication
+# bundle. See https://bugzilla.redhat.com/show_bug.cgi?id=1519323#c2
+rm -rf COPING
+
+%build
+
+%{cmake} . -DBREAKPAD=FALSE
```

Comment:
"-DBREAKPAD=FALSE"
Has been removed upstream

Reply:
Thanks for catching that. I think I had read about breakpad removal and would have thought it was already there but apparently not. Fixed now.